ideascube/serveradmin/management/commands/catalog.py

```diff
@@ -120,7 +124,22 @@ def list_packages(self, options):
             print("Add a remote source with:\n")
             print("    {} remotes add ID NAME URL\n".format(self.parser.prog))
 
-        fmt = ' {0.id:20}  {0.version!s:12}  {0.filesize:8}  {0.name}'
+        fmt = (' {0.id:20}  {0.version!s:12}  {0.filesize:8}'
+               ' {0.type:15} {0.name}')
+
+        unhandled_pkgs = self.catalog.list_nothandled('*')
+        if unhandled_pkgs:
+            if options['filter'] == 'unhandled':
+                print('Not handled packages')
+
+                for pkg in unhandled_pkgs:
+                    print(fmt.format(pkg))
+            else:
+                print(("Warning: There is {} packages ignored by ideascube "
+                       "cause it doesn't know how to handle them.\n"
+                       "Use the command 'list --unhandled' to get the full "
+                       "list of unhandled packages.\n\n"
```

Comment:
Agree.
Used "Warning: Ignoring {} unsupported package(s)\nUse '--unhandled' for details.\n\n" in last push --force